File: lib/singletons/auth_singleton.dart
class AuthSingleton {
  var responseMap = {
    'status': 'requery',
    'message': 'Reaffirm Transaction Status on Server'
  };
  var url = '';

  static final AuthSingleton _singleton = AuthSingleton._internal();

  factory AuthSingleton() {
    return _singleton;
  }

  AuthSingleton._internal();
}

File: android/src/main/kotlin/co/paystack/flutterpaystack/AuthSingleton.kt
package co.paystack.flutterpaystack

/**
 * Created by Wilberforce on 29/07/18 at 06:00.
 */
class AuthSingleton private constructor() {
    var responseJson = "{\"status\":\"requery\",\"message\":\"Reaffirm Transaction Status on Server\"}"
    var url = ""

    companion object {
        val instance = AuthSingleton()
    }

}
